.gitignore

```diff
@@ -1,5 +1,6 @@
 build
 dist
+dist/prebuilds
```

Comment:
Recapping in-person conversation about this: node-gyp was checking the dist folder for the prebuilts instead of the root of the project. We can add an extra .. when we load the addon.